spec/models/reviewable_flagged_post_spec.rb

```diff
@@ -297,12 +297,24 @@ def pending_count
       expect(flagged_reply.reload.status).to eq(Reviewable.statuses[:ignored])
     end
 
-    it 'notifies users that responded to flagged post' do
-      SiteSetting.notify_users_after_responses_deleted_on_flagged_post = true
-      flagged_post.perform(moderator, :delete_and_agree_replies)
+    describe "notifies users that responded to flagged post" do
+      before do
+        SiteSetting.notify_users_after_responses_deleted_on_flagged_post = true
+      end
 
-      expect(Jobs::SendSystemMessage.jobs.size).to eq(2)
-      expect(Jobs::SendSystemMessage.jobs.last["args"].first["message_type"]).to eq("flags_agreed_and_post_deleted_for_responders")
+      it "works" do
```

Comment:
"works" spec description are a no-no. I'd inline the before block in both specs, and drop the `describe` block.

Reply:
😬 whoops